akka-stream-tests/src/test/scala/akka/stream/scaladsl/JsonFramingSpec.scala

```diff
@@ -0,0 +1,444 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.stream.ActorMaterializer
+import akka.stream.impl.JsonBracketCounting
+import akka.stream.scaladsl.Framing.FramingException
+import akka.stream.scaladsl.{ JsonFraming, Framing, Source }
+import akka.stream.testkit.scaladsl.TestSink
+import akka.testkit.AkkaSpec
+import akka.util.ByteString
+import org.scalatest.concurrent.ScalaFutures
+
+import scala.collection.immutable.Seq
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class JsonFramingSpec extends AkkaSpec {
+
+  implicit val mat = ActorMaterializer()
+
+  "collecting multiple json" should {
+    "xoxo parse json array" in {
```

Comment:
hah, good catch - thanks. I use this for a marker for: `testOnly *Json* -- -z xoxo` to only run a specific test, removing